Update getting-started.md (#6129)
kubevirt/kubevirt: docs/getting-started.md

```bash
# Build and deploy KubeVirt on Kubernetes in our vms inside containers

> export KUBEVIRT_PROVIDER=k8s-1.20 # this is also the default if no KUBEVIRT_PROVIDER is set

Reviewer: @yingbaiibm how about removing this line? we will choose the current default anyway. This way we avoid that this line gets outdated.

Author: That's also a choice.   But how about changing the comment to  "Find supported K8s version under KUBEVIRT_TOP/cluster-up/cluster and set required KUBEVIRT_PROVIDER.  If no KUBEVIRT_PROVIDER is set, default value will be used"      In this way, user will be clear about how to set the KUBEVIRT_PROVIDER and choose the version they want.

Reviewer: Works for me too. Anything which does not mention an explicit version (and therefore needing a regular bump) is an improvement.

> export KUBEVIRT_PROVIDER=k8s-1.20 # this is also the default if no KUBEVIRT_PROVIDER is set

Reviewer 2: How about a shorter
```suggestion
# export KUBEVIRT_PROVIDER=k8s-1.20 #  uncomment to use a non-default KUBEVIRT_PROVIDER
```
